server/src/main/java/org/opensearch/indices/RemoteStoreSettings.java

```diff
@@ -65,9 +65,21 @@ public class RemoteStoreSettings {
         Property.Dynamic
     );
 
+    /**
+     * Controls the maximum referenced remote translog files. If breached the shard will be Refreshed.
+     */
+    public static final Setting<Integer> CLUSTER_REMOTE_MAX_TRANSLOG_READERS = Setting.intSetting(
+        "cluster.remote_store.translog.max_readers",
+        300,
+        1,
```

Comment:
Should there be a higher minimum? value of 1 will cause refresh after every write operation. Should we have a rational minimum here?

Reply:
Yes, need to discuss on the minimum and default value here.